spec/layer/identity_token_spec.rb

```diff
@@ -1,22 +1,57 @@
-
+require 'spec_helper'
 
 describe Layer::IdentityToken do
   describe ".new" do
-    it "should allow you to set the user_id, nonce and expires_at variables" do
-      user_id = "1234"
-      nonce = "your_random_nonce"
-      expires_at = "12345678"
+    let(:user_id) { '1234' }
+    let(:nonce) { 'your_random_nonce' }
+    let(:expires_at) { '12345678' }
+    let(:optional_attributes) { {} }
 
-      token = Layer::IdentityToken.new(
+    subject(:token) do
+      Layer::IdentityToken.new(
         user_id: user_id,
         nonce: nonce,
-        expires_at: expires_at
+        expires_at: expires_at,
+        **optional_attributes
```

Comment:
@cakejelly this allows the `optional_parameters` to be passed as keyword arguments instead of as a single hash.